MLS.Agent/CommandLine/CommandLineParser.cs

```diff
@@ -181,6 +181,11 @@ RootCommand StartInTryMode()
                                       "Enable verbose logging to the console",
                                       new Argument<bool>()));
 
+                command.AddOption(new Option(
+                                        "--port",
+                                        "Specify the port to launch the dotnet try process",
```

Comment:
Maybe something a little more casual like: "Specify the port for dotnet try to listen on"?